tests/publiccloud/xfsprepare.pm

```diff
@@ -39,7 +39,9 @@ sub install_xfstests {
     ensure_user_exists("daemon", 2);
     # Create test users (See https://git.kernel.org/pub/scm/fs/xfs/xfstests-dev.git/tree/README)
     assert_script_run("useradd -mU fsgqa");    # Create home directory (-m) and 'fsgqa' group for the user (-U) as well
-    script_run("useradd 123456-fsgqa");    # script_run because only required by few tests and there is a chance that users starting with digits won't work
+
+    # The following is only required by few tests and there is a chance that users starting with digits won't work
+    record_info("123456-fsgqa", "error creating 123456-fsgqa user.\nSome tests will not be able to run") if (script_run("useradd 123456-fsgqa") != 0);
```

Comment:
I somehow don't like this format where you actually do (create?) something important. I'd do something like:
```suggestion
    script_run("useradd 123456-fsgqa");
    record_info("123456-fsgqa", "error creating 123456-fsgqa user.\nSome tests will not be able to run") if (script_run("cat /etc/passwd | grep 123456-fsgqa") != 0);
```

Reply:
I agree but I think will be better to use assert_script_run in line 44

`assert_script_run("useradd 123456-fsgqa")`

Reply:
We can't assert this, as this fails on our SLES images. I still leave it there, as it is recommended to have this user, but one can see it as a note, that we are not able to run some tests due to this limitation.

Reply:
Please check again, I've updated the corresponding lines.

Reply:
understood, thanks for the explanation

Reply:
You're welcome :slightly_smiling_face:

Reply:
👍